packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/InputAwareWebView.java

```diff
@@ -22,16 +24,29 @@
  * <p>See also {@link ThreadedInputConnectionProxyAdapterView}.
  */
 final class InputAwareWebView extends WebView {
-  private final View containerView;
-
+  private static final String TAG = "InputAwareWebView";
   private View threadedInputConnectionProxyView;
   private ThreadedInputConnectionProxyAdapterView proxyAdapterView;
+  private @Nullable View containerView;
 
-  InputAwareWebView(Context context, View containerView) {
+  InputAwareWebView(Context context, @Nullable View containerView) {
     super(context);
     this.containerView = containerView;
   }
 
+  void setContainerView(@Nullable View containerView) {
+    this.containerView = containerView;
+
+    if (proxyAdapterView == null) {
+      return;
+    }
+
+    Log.e(TAG, "The containerView has changed while the proxyAdapterView exists.");
+    if (containerView != null) {
+      setInputConnectionTarget(proxyAdapterView);
```

Comment:
Yeah, this does work from my testing. Semantically this seems "wrong" to me, because we're changing the view after it's already been launched. I'm worried that there may be other bugs under the hood. But it is spammy to log this without a confirmed error. I downgraded to `WARNING`.